banzai/images.py

```diff
@@ -131,6 +131,40 @@ def write_catalog(self, filename, nsources=None):
     def add_history(self, msg):
         self.header.add_history(msg)
 
+    def data_is_3d(self):
+        try:
+            data_is_3d = len(self.data.shape) > 2
+        except AttributeError:
+            logger.error("Trying to measure dimensions of data without a shape attribute",
+                          extra={'tags': {'filename': self.filename}})
+            data_is_3d = False
+        return data_is_3d
+
+    def get_n_amps(self):
+        if self.data_is_3d():
+            n_amps = self.data.shape[0]
+        else:
+            n_amps = 1
+        return n_amps
+
+    def get_inner_image_section(self, edge_fraction_denominator=4):
```

Comment:
edge_fraction_denominator is a bit of a confusing name. How about something like section_denominator?

Reply:
I changed it to multiply `nx` and `ny` and renamed it `fractional_edge_width`.